common/web/gesture-recognizer/src/pathSegmentStats.ts

```diff
@@ -0,0 +1,345 @@
+namespace com.keyman.osk {
+  export class PathSegmentStats {
```

Comment:
Note to self - maybe call it `CumulativePathStats` or something.  (The name was set kinda early into my experimentations that led to the current code.)